lib/pbench/cli/agent/commands/tools/list.py

```diff
@@ -0,0 +1,142 @@
+"""
+pbench-list-tools
+
+This script can list all tools from all groups, list tools from a specific
+group, or list which gourps contain a specific tool
+
+"""
+
+import logging
+import sys
+
+import click
+
+from pbench.agent.utils import setup_logging
+from pbench.cli.agent import CliContext, pass_cli_context
+from pbench.cli.agent.commands.tools.base import ToolCommand
+from pbench.cli.agent.options import common_options
+
+LOG = logging.getLogger(__name__)
```

Comment:
Is there a requirement that this be global?

Can you move this to the constructor for`ListTools`?